src/web/handlers/PanGestureHandler.ts

```diff
@@ -299,11 +314,15 @@ export default class PanGestureHandler extends GestureHandler {
     super.onPointerMove(event);
   }
 
-  protected onPointerOutOfBounds(event: AdaptedEvent): void {
+  protected onPointerOutOfBounds(event: AdaptedEvent, ev: TouchEvent): void {
```

Comment:
Same as above

Reply:
Done in [5c26a86](https://github.com/software-mansion/react-native-gesture-handler/pull/2631/commits/5c26a86fd4ccfd2a17e2d54e49d54a50c5b2843a) and [cf5e1cc](https://github.com/software-mansion/react-native-gesture-handler/pull/2631/commits/cf5e1cc89a8febd6e07649c7e3af630d238e1c86)